src/status_im2/contexts/add_new_contact/views.cljs

```diff
@@ -5,7 +5,6 @@
     [react-native.core :as rn]
```

Comment:
Hi @erikseppanen, this screen should use bottom-sheet-screen https://github.com/status-im/status-mobile/pull/15399

Reply:
done